lib/game_box_web/components/arena_code.ex

```diff
@@ -0,0 +1,35 @@
+defmodule GameBoxWeb.ArenaCode do
+  @moduledoc """
+  A component for displaying the Arena Code and a copy link.
+  """
+  use Phoenix.Component
+
+  import GameBoxWeb.Typography
+  alias Phoenix.LiveView.JS
+
+  def arena_code(assigns) do
+    ~H"""
+    <.h5 class="!text-primary !pt-0 !mt-0" label="Arena" />
+    <.h1 :if={@arena_id} label={@arena_id} />
+
+    <.p class="text-sm">
+      Have your friends enter this code at gamebox.fly.dev to join!
+    </.p>
+
+    <div>
+      <.p class="text-secondary text-sm !m-0">
+        Or
+        <a
+          class="cursor-pointer underline text-primary hover:text-white"
+          phx-click={JS.dispatch("gamebox:clipcopy", to: "#arena-code")}
+        >
+          copy an invite link
+        </a>
+        <div class="copied hidden text-xs italic">Copied!</div>
+      </.p>
+
+      <input type="hidden" id="arena-code" value={"gamebox.fly.dev/join?arena=#{@arena_id}"} />
```

Comment:
I think here's where you could use a `Routes.live_path()` to generate the path with the correct host

Reply:
@nschello I tried this 
```
value={"#{Routes.live_path(GameBoxWeb.Endpoint, GameBoxWeb.HomeLive)}/join?arena=#{@arena_id}"}
```

But the URL it then copies is //join?arena=1234. It's like it grabs the wrong end of the path. Am I missing another way to use it?

Reply:
Looks like I can access it through a `handle_params`. I'll add that and tag you again.